packages/ember-glimmer/lib/component-managers/custom.ts

```diff
@@ -0,0 +1,101 @@
+import { assert } from 'ember-debug';
+import AbstractManager from './abstract';
+import {
+  CurlyComponentLayoutCompiler
+} from './curly';
+import { ComponentManager } from '@glimmer/runtime';
+import ComponentStateBucket from '../utils/curly-component-state-bucket';
+import { ComponentCapabilities } from './capabilities';
+
+/**
+  Wrapper class for custom component managers as per
+  Custom Components RFC.
+
+  When defining a custom component manager, the following
+  methods are required:
+
+  - `create`
+  - `getSelf`
+  - `update`
+
+  `layoutFor` method is not required to be implemented and
```

Comment:
This comment infers that it isn't required to be implemented, but it doesn't explain if it _can_ be implemented?